draft-packages/modal/KaizenDraft/Modal/Presets/ContextModal.tsx

```diff
@@ -15,9 +15,11 @@ export type InformationModalSecondaryActionProps =
       secondaryLabel?: undefined
     }
 
-export type InformationModalProps = Readonly<
+export type ContextModalProps = Readonly<
   {
     isOpen: boolean
+    unpadded?: boolean
+    isLandscape?: boolean
```

Comment:
Just an fyi: I'd have gone with a `layout` prop that takes `"portrait"|"landscape"` (with "portrait" as the default), as it's a bit more expressive IMO than a boolean and allows for more extensibility if something like "round" or "stretched" ever came up (fwiw I don't think they will).

Reply:
Done :)